src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -59,6 +59,7 @@ public EventHistory buildEvents(CaseData caseData) {
                 FlowState.Main flowState = (FlowState.Main) FlowState.fromFullName(state.getName());
                 switch (flowState) {
                     case TAKEN_OFFLINE_UNREPRESENTED_DEFENDANT:
+                    case TAKEN_OFFLINE_UNREPRESENTED_UNREGISTERED_DEFENDANT:
```

Comment:
Same as previous, one to keep an eye on for future refactoring (only if we need to change the wording)